.github/workflows/linters.yml

```diff
@@ -0,0 +1,38 @@
+name: Python linters
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  linters:
+    name: Run Python linters
+    runs-on: ${{ matrix.os }}
+
+    strategy:
+      matrix:
+        python-version: [3.8]
+        os:
+          - ubuntu-latest
+
+    defaults:
+      run:
+        shell: bash
+
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-python@v1
+        with:
+          python-version: ${{ matrix.python-version }}
+      - name: Install dependencies
+        run: |
+          python -m pip install --upgrade pip
+          pip install -e .
+          pip install -r examples/python-shell/provider/requirements.txt
+          pip install mypy==0.910 pylint==2.9.*
```

Comment:
I think it would be better to install the latest version here. I don't think this is important though.

Reply:
This may lead to some unpleasant situations, e.g. someone changes a line in README and suddenly gets an ❌ because a new version of PyLint has been released since the previous commit to master.

Reply:
OK, that makes sense.

I just don't believe "we just have to remember to update this on regular basis" and I'm concerned we'll end up using current mypy/pylint longer than we should, e.g. forever.

Reply:
I'll freeze the major version only.